post_office/management/commands/load_email_template.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from ...models import EmailTemplate
+from ...logutils import setup_loghandlers
+
+logger = setup_loghandlers()
+
+
+def _get_template(template_name):
+    """Find the template and return the raw template file content"""
+
+    template = loader.get_template(template_name)
+    content = template.template.source
+    content = content.strip()
```

Comment:
I added it because I had an issue of subject.txt having a `\n` in the end, so it fails when post_office tries to send the actual email.

Will `title.replace('\n', '')` be better?